src/model/model.nit

```diff
@@ -722,6 +722,11 @@ class MClassDef
 
 	# All property definitions in the class (introductions and redefinitions)
 	var mpropdefs = new Array[MPropDef]
+
+	# All property definitions in the class by its associated property.
+	#
+	# This include introductions and redefinitions, not inheritance.
+	var mpropdefs_by_property = new HashMap[MProperty, MPropDef]
```

Comment:
Not sure for the documentation. For what I see, this dictionary only includes `MPropDef`s of the class definition, not of the entire class.